### ISSUE TITLE
t/MARC-Convert-Wikidata-Object/08-compilers.t started to fail

### ISSUE THREAD

**Issue by eserte** [michal-josef-spacek/MARC-Convert-Wikidata-Object#1]
The test suite fails on my smokers:
```
#   Failed test 'Get default compilers list.'
#   at t/MARC-Convert-Wikidata-Object/08-compilers.t line 12.
#     Structures begin differing at:
#          $got = undef
#     $expected = ARRAY(0x804250630)
# Looks like you failed 1 test of 3.
t/MARC-Convert-Wikidata-Object/08-compilers.t ................ 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/3 subtests 
```

See also http://matrix.cpantesters.org/?dist=MARC-Convert-Wikidata-Object;reports=1#sl=7,1 --- the problem started in August 2023, but not all reports since then are fail reports.

I suspect that a change in Mo-utils could be responsible, maybe this one: https://metacpan.org/release/SKIM/Mo-utils-0.20/source/Changes#L12

**Comment by michal-josef-spacek** [comment]
@eserte Ha, thanks for report.

**Comment by michal-josef-spacek** [comment]
Fixed in 0.02 release
